content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -45,17 +46,21 @@ including the `--oauth-no-pkce` flag when starting `chronograf`.
 - Flux improvements and additional functionality:
   - Add dashboard template variables to Flux query execution.
     Flux queries include a `v` record with a key value pair for each variable.
-  - Support dashboard template variables defined with Flux.
-  - Add Kapacitor Flux Tasks on the Manage Tasks page (read only).
+  - Support dashboard [template variables defined with Flux](/chronograf/v1.9/guides/dashboard-template-variables/#flux-query).
+  - Add [Kapacitor Flux Tasks](/chronograf/v1.9/guides/advanced-kapacitor/#manage-kapacitor-flux-tasks)
```

Comment:
```suggestion
  - Add [Kapacitor Flux tasks](/chronograf/v1.9/guides/advanced-kapacitor/#manage-kapacitor-flux-tasks)
```

Reply:
Add support for Flux tasks in Kapacitor ...or are we literally adding specific Flux tasks?

Reply:
It adds a section to the Manage tasks page that lists Flux tasks added to Kapacitor.